Rubberduck.Core/UI/DockableToolwindowPresenter.cs

```diff
@@ -68,7 +68,8 @@ private IWindow CreateToolWindow(IDockableUserControl control)
                 throw;
             }
 
-            var userControlHost = (_DockableWindowHost)_userControlObject;
+            // could cast to COM_IOleWindow for the first few, but AddUserControl is only on _DockableWindowHost
+            dynamic userControlHost = _userControlObject;
```

Comment:
I think the comment should be more clearer --- few what? Why those but not that?

Reply:
instructions, in this case. Or member accesses....

Reply:
Ignore that... apparently I'm just rather incapable of properly reading the code...